README.md

```diff
@@ -28,21 +28,39 @@ The repo includes sample data so it's ready to try end to end. In this sample ap
 - Git (install from [here](https://git-scm.com/downloads))
 - Powershell (pwsh) (install from [here](https://github.com/powershell/powershell))
    - **Imporant**: Ensure you can run pwsh.exe from a PowerShell command. If this fails, you likely need to upgrade PowerShell.
+- Your Azure Account must have `Microsoft.Authorization/roleAssignments/write` permissions, such as [User Access Administrator](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles#user-access-administrator) or [Owner](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles#owner).  
 
 ### Installation
 
-Starting from scratch:
+#### Starting from scratch:
 1. Create a new folder and switch to it in the terminal
-2. Run "azd up -t azure-search-openai-demo"
-    * For the target location, the regions that currently support the models used in this sample are East US or South Central US. For an up-to-date list of regions and models, check [here](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/concepts/models)
+2. Run `azd up -t azure-search-openai-demo`
+    * For the target location, the regions that currently support the models used in this sample are **East US** or **South Central US**. For an up-to-date list of regions and models, check [here](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/concepts/models)
 
-Deploying or re-deploying a local clone of the repo:
-* Simply run "azd up"
+#### Use existing resources:
+1. Create a new folder and switch to it in the terminal
+1. Run `azd init -t azure-search-openai-demo`
+1. Run `azd env set AZURE_OPENAI_RESOURCE_GROUP {NAME OF EXISTING RESOURCE GROUP}`
+1. Run `azd env set AZURE_OPENAI_SERVICE {NAME OF EXISTING OPEN AI SERVICE}`
```

Comment:
For completeness, if we're pointing to an existing OpenAI service, should we show how to set the name of the OpenAI deployments?

Reply:
IC, yes, I can do that.